mapnik_c_api.cpp

```diff
@@ -255,6 +269,15 @@ mapnik_image_blob_t * mapnik_image_to_png_blob(mapnik_image_t * i) {
     return blob;
 }
 
+const char * mapnik_version_string() {
+#if MAPNIK_VERSION >= 200100
+	return MAPNIK_VERSION_STRING;
+#else
+#define MAPNIK_C_API_STRINGIFY(n) #n
+	return "ABI " MAPNIK_C_API_STRINGIFY(MAPNIK_VERSION);
+#endif
+}
```

Comment:
This looks really odd. What are you trying to accomplish? Why put an `ABI ` in front when using more recent mapnik? Also `mapnik/config.hpp` does not include any C++ so it can be safely included by C apps.